content/docs/dvclive/api-reference/live/index.md

```diff
@@ -62,11 +62,14 @@ other metadata.
 
   </admon>
 
-- `report` - If `html`, DVCLive will call `Live.make_report()` on each step
-  update. _Default_: `html`.
+- `report` - If `auto`,`html` or `md`, DVCLive will call `Live.make_report()` on
+  each step update. _Default_: `auto`.
 
-- `auto_open` - If `True`, on the first `Live.make_report()` call, DVCLive will
-  automatically open `html_path` in a browser. _Default_: `False`.
```

Comment:
Wait what happened to `auto_open`?

Reply:
Docs were outdated. I just noticed this was still here during this P.R. 
It was removed a few versions ago, discussion here https://github.com/iterative/dvclive/issues/241